.gitignore

```diff
@@ -7,12 +7,7 @@
 *.log
 *.log.*
 
-# Compilation artifacts
-src/mixxx.rc.include
-src/mixxx.res
-src/test/**/*.actual
-res/qrc_mixxx.cc
-
+# TODO: Do we really need this exclude?
```

Comment:
Yes, this may be created if you trun a python tool from the tools directory.

Reply:
Ok, I'll modify this commit and force push.